optimize/components/userguide/processes.md

```diff
@@ -7,7 +7,7 @@ description: "The processes section gives you an overview of all the processes a
 ## Overview
 
 The processes section gives you an overview of all the processes and their status on a single page.
```

Comment:
The section is not called processes anymore. I suggest adjusting this to something like 'Process dashboards' or move the documentation to be inside the dashboards section. It might be a good idea to discuss this with @toco-cam as well.